monitoring.go

```diff
@@ -49,6 +49,7 @@ func launchMonitoringServer(conf *config.Config) {
 		}
 	})
 
+	http.HandleFunc("/quit", quitHandler)
```

Comment:
Minor nit: I'm now thinking if this should be /exit for consistency with the os.Exit action? I don't feel strongly about this so up to you!